host/xtest/pkcs11_1000.c

```diff
@@ -66,19 +67,28 @@ static void xtest_tee_test_1001(ADBG_Case_t *c)
 	if (!ADBG_EXPECT_CK_OK(c, rv))
 		return;
 
+	Do_ADBG_BeginSubCase(c, "Test C_GetFunctionList()");
+
 	rv = C_GetFunctionList(&ckfunc_list);
 	if (!ADBG_EXPECT_CK_OK(c, rv))
 		goto out;
 
 	if (!ADBG_EXPECT_NOT_NULL(c, ckfunc_list->C_GetInfo) ||
 	    !ADBG_EXPECT_NOT_NULL(c, ckfunc_list->C_GetSlotList) ||
-	    !ADBG_EXPECT_NOT_NULL(c, ckfunc_list->C_GetSlotInfo))
+	    !ADBG_EXPECT_NOT_NULL(c, ckfunc_list->C_GetSlotInfo) ||
+	    !ADBG_EXPECT_NOT_NULL(c, ckfunc_list->C_GetTokenInfo))
 		goto out;
 
+	Do_ADBG_EndSubCase(c, NULL);
```

Comment:
It is valid to end a sub-case like this, but with a string it's a bit easier when reading the code to know which sub-case is ended.

Reply:
ok, will do.